src/dbnode/storage/series/buffer.go

```diff
@@ -272,11 +272,6 @@ func (b *dbBuffer) Write(
 		writeType   WriteType
 	)
 	switch {
-	case wOpts.BootstrapWrite:
```

Comment:
Does this mean bootstrap writes will be classified as cold writes?

Reply:
I changed this so that basically whether the writes are for file volumes that already exist or not the bootstrap writes will be considered a cold write or not. 

This I believe reflects accurately what type of write it is when it's being accumulated from the commit log.